src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/UnusedTagsPass.php

```diff
@@ -75,6 +75,7 @@ class UnusedTagsPass implements CompilerPassInterface
         'routing.expression_language_provider',
         'routing.loader',
         'routing.route_loader',
+        'routing.service',
```

Comment:
the name seems a little too generic. maybe "routing.expression_language_service" or "routing.condition_service" (as it's for the routing condition)?